### PR TITLE
change hardcode key to os variables

### LINKED ISSUE #45: Create developer package of docker
Create developer package of docker

### PR DESCRIPTION
**Pull request by shahryarjb**

### CODE REVIEW

File: config/config.exs [hunk 1]
```diff
@@ -55,11 +53,27 @@ config :mishka_user, MishkaUser.Guardian,
   allowed_algos: ["HS256"],
   secret_key: %{
   "alg" => "HS256",
+  # "k" => "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9",
   "k" => "Exe6Qk6YPWWNmOS7rAtXQfPPngruPtEIivDB1nsXwSk",
```
Comment (shahryarjb) [inline config/config.exs:57]: Need to attention and create with shell command!! @mojtaba-naserei

File: apps/mishka_database/priv/repo/seeds.exs [hunk 2]
```diff
@@ -5,143 +5,137 @@ alias MishkaContent.Blog.TagMapper
...
-  case Tag.create(%{
-    title: "برچسب #{x}",
-    alias_link: "barchasb-#{x}",
-    meta_keywords: "برچسب #{x}",
-    meta_description: "این برچسب در مورد برچسب #{x} هست",
-    custom_title: "برچسب #{x}",
-    robots: :IndexFollow,
-    }) do
-    {:ok, :add, :blog_tag, tag_info} -> tag_info.id
+if User.users(conditions: {1, 5}, filters: %{}).entries == [] do
```
Comment (shahryarjb) [inline apps/mishka_database/priv/repo/seeds.exs:8]: we check seeds if this loaded or not

File: apps/mishka_database/priv/repo/seeds.exs [hunk 3]
```diff
@@ -5,143 +5,137 @@ alias MishkaContent.Blog.TagMapper
...
-  "title" => "جوملا",
-  "short_description" => "Test category description",
-  "main_image" => "/uploads/a2fb7e66-c9c6-4d9a-8739-6c26a7b927ff.jpg",
-  "description" => "این یک مجموعه برای صحبت در مورد موضوع جوملا می باشد.",
-  "alias_link" => "جوملا",
-  },
-  %{
-    "title" => "طراحی سایت",
-    "short_description" => "Test category description",
-    "main_image" => "/uploads/a2fb7e66-c9c6-4d9a-8739-6c26a7b927ff.jpg",
```
Comment (shahryarjb) [inline apps/mishka_database/priv/repo/seeds.exs:64]: all test images should be changed and compress

File: apps/mishka_database/priv/repo/seeds.exs [hunk 4]
```diff
@@ -5,143 +5,137 @@ alias MishkaContent.Blog.TagMapper
...
+    "status" => 1,
+    "unconfirmed_email" => nil
+  }
+
+  tags = Enum.map(Enum.shuffle(1..10), fn x ->
+    case Tag.create(%{
+      title: "برچسب #{x}",
+      alias_link: "barchasb-#{x}",
+      meta_keywords: "برچسب #{x}",
+      meta_description: "این برچسب در مورد برچسب #{x} هست",
```
Comment (mojtaba-naseri) [inline apps/mishka_database/priv/repo/seeds.exs:23]: after localization we should create a better seeds to test multi languages

File: config/config.exs [hunk 5]
```diff
@@ -55,11 +53,27 @@ config :mishka_user, MishkaUser.Guardian,
...
   "kty" => "oct",
   "use" => "sig"
 }
 
+config :mishka_content, MishkaContent.Email.Mailer,
+  adapter: Bamboo.LocalAdapter
```
Comment (mojtaba-naseri) [inline config/config.exs:63]: before releasing  v0.0.1 we should change test email config and its test router

**Pull request merged**